Tasks/DeployAzureResourceGroup/SPNCreation.ps1

```diff
@@ -14,20 +14,21 @@ param
 $ErrorActionPreference = "Stop"
 $VerbosePreference = "SilentlyContinue"
 $userName = $env:USERNAME
-$newguid = New-Guid
-$displayName = "VSO." + $userName + '.' + $subscriptionName + '.' + $newguid
+$newguid = [guid]::NewGuid()
+$displayName = [String]::Format("VSO.{0}.{1}.{2}", $userName, $subscriptionName.Replace(' ', ''), $newguid)
 $homePage = "http://" + $displayName
 $identifierUri = $homePage
 
 
 #Initialize subscription
-$isAzureModulePresent = Get-Module -Name AzureRM -ListAvailable
+$isAzureModulePresent = Get-Module -Name AzureRM* -ListAvailable
 if ([String]::IsNullOrEmpty($isAzureModulePresent) -eq $true)
 {
-    Write-Output "Script requires AzureRM module to be present. Obtain AzureRM from https://www.powershellgallery.com/packages/AzureRM/" -Verbose
+    Write-Output "Script requires AzureRM module to be present. Obtain AzureRM from https://github.com/Azure/azure-powershell/releases/tag/v1.0.2-December2015" -Verbose
```

Comment:
Do you want the link specifically to be 1.0.2? If so then, every time we update our support we have to keep updating this link in script. Should you just point to the parent link listing all updates and add text that refer to our readme on supported versions.